transip/service/dns.py

```diff
@@ -0,0 +1,50 @@
+"""
+DNS related objects
+"""
+
+from suds.sudsobject import Object as SudsObject
+
+
+class DnsEntry(SudsObject):
+    """
+    Representation of the DnsEntry object
+    """
+    TYPE_A = 'A'
+    TYPE_AAAA = 'AAAA'
+    TYPE_CNAME = 'CNAME'
+    TYPE_MX = 'MX'
+    TYPE_NS = 'NS'
+    TYPE_TXT = 'TXT'
+    TYPE_SRV = 'SRV'
+
+    name = None
+    expire = 0
+    type = None
+    content = None
+
+    def __init__(self, name, expire, type, content):
```

Comment:
the parameter `type` will mask the [build in function](https://docs.python.org/2/library/functions.html#type)

proposed: `entry_type` or `record_type`